### PR TITLE
Added support for the BTCe exchange

### PR DESCRIPTION
**Pull request by tuxitor**
Warning - Some of the functions need testing by someone in posession of a BTCe account. In particular this is the case for  the buy(), sell(), cancelOrder() and getOrder() functions

### CODE REVIEW

File: extensions/exchanges/btce/exchange.js [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+      var client = authedClient()
+      client.getInfo(function (err, body) {
+        body = statusErr(err, body)
+        if (err) {
+          return retry('getBalance', func_args, err)
+        }
+        if (body.success) {
+          var balance = {asset: 0, currency: 0}
+          var funds = body.return.funds
+          balance.currency = 999.450009 //funds[args.currency]
```
Comment (nedievas) [inline extensions/exchanges/btce/exchange.js]: ?

File: extensions/exchanges/btce/exchange.js [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+          balance.asset = 0.450008 //funds[args.asset]
```
Comment (nedievas) [inline extensions/exchanges/btce/exchange.js]: ?

**Comment by DeviaVir** [comment]
@MrSiNiSt3R might be interested in testing

**Comment by crubb** [comment]
https://www.justice.gov/usao-ndca/pr/russian-national-and-bitcoin-exchange-charged-21-count-indictment-operating-alleged <- Hmm... ;)

**Review by DeviaVir — APPROVED** [review]
:+1: great work!

**Pull request merged**